kv-components/vue/KvLoadingSpinner.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+	<svg
+		class="animate-spin"
+		:class="`${cssClassSize} ${cssClassColor}`"
+		fill="none"
+		xmlns="http://www.w3.org/2000/svg"
+		viewBox="0 0 24 24"
+	>
+		<g
+			clip-path="url(#clip0)"
+			stroke-width="2"
+		>
+			<circle
+				class="opacity-low"
+				cx="12"
+				cy="12"
+				r="8"
+				transform="rotate(135 12 12)"
+				stroke="#C4C4C4"
+			/>
+			<path
+				d="M17.447 17.866c3.24-3.24 3.334-8.399.21-11.523-3.125-3.124-8.284-3.03-11.524.21"
+				stroke="currentColor"
+				stroke-linecap="round"
+			/>
+		</g>
+		<defs>
+			<clipPath id="clip0">
+				<path
+					fill="#fff"
+					d="M0 0h24v24H0z"
+				/>
+			</clipPath>
+		</defs>
+	</svg>
+</template>
+
+<script>
+/**
+ * Animated SVG Loading Spinner with variable size and color.
+ */
+
+export default {
+	props: {
+		/**
+		 * The size of the loading spinner.
+		 * 'small', 'medium', or 'large'
+		 * */
+		size: {
+			type: String,
+			default: 'medium',
+			validator(value) {
+				return ['small', 'medium', 'large'].includes(value);
+			},
+		},
+		/**
+		 * The color of the loading spinner.
+		 * 'brand', 'white', or 'black'
+		 * */
+		color: {
```

Comment:
Not sure if we want this to be a prop. Alternatively, you can just add the text-* class to the component. Although it does seem nice to limit the available options of colors to just these 3.

Reply:
I think the prop and limiting these is good right now 👍